rest-api-spec/schema.json

```diff
@@ -0,0 +1,242 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
curious to why draft-07 and not [2019-09](https://json-schema.org/draft/2019-09/release-notes.html) (formerly draft-08) ?

I think the difference here are mostly cosmetic, so probably not a big deal either way. just curious if there was a specific reason ?

Reply:
Used draft-07 as it was the latest version that the JavaScript validator the clients team were using supported